cmd/convox/install_test.go

```diff
@@ -16,12 +16,52 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var installUsage = `convox install: install convox into an aws account
```

Comment:
I want to test:

- that usage is printed when requested
- that usage is printed when there are invalid, missing or extra args
- that missing credentials files are handled
- that hostname resolution failures and "no such file or directory" errors are handled as expected
- that we don't panic in any case

I would be ok with a less strict usage test; for now I was just following the same pattern as in #1996.